src/current/cockroachcloud/provision-a-cluster-with-terraform.md

```diff
@@ -19,56 +19,44 @@ This tutorial shows you how to provision a CockroachDB Cloud cluster using the C
 Before you start this tutorial, you must
 
 - [Install Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli).
-- Install the [`wget` command line utility](https://www.gnu.org/software/wget/).
-- Create a [service account]({% link cockroachcloud/managing-access.md %}#manage-service-accounts) and [API key]({% link cockroachcloud/managing-access.md %}#api-access) in the [CockroachDB Cloud Console](https://cockroachlabs.cloud), and assign `admin` privilege or Cluster Creator / Cluster Admin role at the organization scope. Refer to: [Service Accounts]({% link cockroachcloud/authorization.md %}#service-accounts)
+- Create a [service account]({% link cockroachcloud/managing-access.md %}#manage-service-accounts) and [API key]({% link cockroachcloud/managing-access.md %}#api-access) in the [CockroachDB Cloud Console](https://cockroachlabs.cloud), and assign the Cluster Creator role at the organization scope. Refer to: [Service Accounts]({% link cockroachcloud/authorization.md %}#service-accounts)
 
 ## Create the Terraform configuration files
 
 Terraform uses a infrastructure-as-code approach to managing resources. Terraform configuration files allow you to define resources declaratively and let Terraform manage their lifecycle.
 
 <section class="filter-content" markdown="1" data-scope="serverless">
 
-In this tutorial, you will create a CockroachDB {{ site.data.products.serverless }} cluster.
-
-1. In a terminal create a new directory and use `wget` to download the CockroachDB {{ site.data.products.serverless }} `main.tf` example file:
+1. Download the CockroachDB {{ site.data.products.serverless }} `main.tf` example file:
```

Comment:
There's some issue here with rendering up to the Provision the cluster heading. I think it may be due to misaligned indentation